__test__/api/createSingleOrganization.test.js

```diff
@@ -0,0 +1,21 @@
+const app = require("../../server");
+const supertest = require("supertest");
+const request = supertest(app);
+
+describe("POST /organization", () => {
+  test("Should create a new organization", async () => {
+    const res = await request.post("/v1/organizations").send({
+      organizationName: "HNG",
+      organizationEmail: "hng@email.com",
+      secret: "dohrehmifasolati",
+      adminName: "John Doe",
+      adminEmail: "johndoe@gmail.com",
+      adminPassword: "adminpassword",
+    });
+
+    expect(res.status).toBe(201);
+    expect(res.body.data.organizationId).toBeTruthy();
```

Comment:
change all `truthy` assertions to match real values

Reply:
All values are required. The first 3 are necessary to create an organization and get the organizationId. The next 3 are necessary to create a default admin user and get the adminId. Both Id(s) are required to create the token. So you can say if any of the parameters are void, it will throw this 422 error